lib/resources/r4/financial_transaction_create.yaml

```diff
@@ -0,0 +1,288 @@
+---
+name: Financial Transaction
+field_name_base_url: http://hl7.org/fhir/r4/Basic
+fields:
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The type of the FHIR resource.
+  action: create
+  example: |
+    {
+      "resourceType":"Basic"
```

Comment:
nit: add a space after the colon

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/4adf16f12daf8a74d8b1caef1c28f22dd3d0b8fd#diff-26870e1b94fc9955e130f3f940dbe2e7R13

Reply:
+1